packages/react-examples/src/react/FocusTrapZone/e2e/FocusTrapZone.Empty.stories.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import { FocusTrapZone, IFocusTrapZoneProps, mergeStyles } from '@fluentui/react';
+import type { FTZTestWindow } from './shared';
+
+const rootClass = mergeStyles({
+  button: { height: 30, width: 60 },
+});
+
+/**
+ * Tab and shift-tab do nothing (keep focus where it is) when the FTZ contains 0 tabbable items
+ */
+export const Empty = () => {
+  const [props, setProps] = React.useState<IFocusTrapZoneProps | undefined>();
+
+  React.useEffect(() => {
+    (window as FTZTestWindow).setProps = setProps;
```

Comment:
I honestly don't think this is bad, I guess a more 'proper' way would be to use storybook to manage the globals

Reply:
Do you know how to manage globals with storybook? I tried setting up the story to accept parameters, which can be set in the query, but IIRC the issue with that was `cy.loadStory` didn't have an API to set query params.

Reply:
@Hotell  and wrote some code to handle globals for the theme picker in the v9 storybook docs, you can take a look in `react-storybook-addon`, but you don't need to do query params to manage storybook globals.

That being said, SB globals can be ugly to manage, that's why I said I honestly don't think it's that bad

Reply:
FYI: All my local `--mode open` runs with the previous approach (setting a global but not cleaning it up) worked fine, but a lot of the tests failed in headless mode in the PR due to a timing issue. I guess because runs in a browser UI are slower, each run always got the current test's version of `setProps`, but in headless runs, a lot of times the test would start running and call the previous test's `setProps` before the story had time to finish rendering and set a new one.

Fix is:
- define the global inside a one-time effect WITH cleanup that deletes the global
- in the test, use `window.should` not `window.then` to wrap the `setProps` call, so that it retries if the global isn't ready yet